loading/functions.js

```diff
@@ -0,0 +1,29 @@
+const { productNumberRange } = require('./testConfig.js');
+
+// GET PUT DELETE @ /description/:productId routes
+// sets random productId params
+function setProductId (requestParams, context, ee, next) {
+  const randomProductNumber = Math.floor(Math.random() * productNumberRange);
+  const newUrl = `/description/${randomProductNumber}`;
+  requestParams.url = newUrl;
+  return next()
+}
+
```

Comment:
If I understand correctly, this function is to update the request url as it comes in on each request. I would have thought that the testing library would have the functionality built in to do something like this. Did you build this because you didn't see anything like that?

Reply:
There probably is but i didnt find anything that  randomized variables without hardcoding them beforehand, so this was a quick hackish solution